internal/kuberesource/sets.go

```diff
@@ -546,6 +547,9 @@ func Emojivoto(smMode serviceMeshMode) []any {
 		smIngressConfigAnnotationKey: "web#8080#false",
 		smEgressConfigAnnotationKey:  "emoji#127.137.0.1:8081#emoji-svc:8080##voting#127.137.0.2:8081#voting-svc:8080",
 	})
+	voteBot.WithAnnotations(map[string]string{
+		smEgressConfigAnnotationKey: "web#127.137.0.3:8081#web-svc:443",
+	})
```

Comment:
not sure if this makes sense. The vote bot isn't part of the confidential deployment, it simulates a user accessing the service. Why are we setting a service mesh annotation on it? It shouldn't have a contrast runtime class.

Reply:
Oh, that's interesting - this is not what's currently implemented. None of our resources have a `runtimeClassName` set, e.g. 

https://github.com/edgelesssys/contrast/blob/e115304b5d273e1cbff981e142b7ed71e0293580/internal/kuberesource/sets.go#L286-L312

Instead, the runtime class is added globally with

https://github.com/edgelesssys/contrast/blob/e115304b5d273e1cbff981e142b7ed71e0293580/e2e/servicemesh/servicemesh_test.go#L44

which is why the egress annotation here works.

Reply:
As discussed in private, we're keeping the bot in the mesh in order to have more test targets.